### PR TITLE
Remove non-functional Transforms from presets

### PR DESCRIPTION
**Pull request by datumbox**
Fixes partially #4652

cc @datumbox @vfdev-5 @bjuncek

### CODE REVIEW

File: torchvision/prototype/models/video/resnet.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def _video_resnet(
 class R3D_18Weights(Weights):
     Kinetics400_RefV1 = WeightEntry(
         url="https://download.pytorch.org/models/r3d_18-b3b3357e.pth",
-        transforms=partial(Kinect400Eval, resize_size=(128, 171), crop_size=(112, 112)),
+        transforms=partial(Kinect400Eval, crop_size=(112, 112), resize_size=(128, 171)),
```
Comment (datumbox) [inline torchvision/prototype/models/video/resnet.py:65]: Changing order to match the other presets.

File: torchvision/prototype/transforms/_presets.py [hunk 2]
```diff
@@ -26,42 +25,47 @@ def __init__(
...
     def forward(self, img: Tensor) -> Tensor:
-        img = self._crop(self._resize(img))
+        img = F.resize(img, self._size, interpolation=self._interpolation)
+        img = F.center_crop(img, self._crop_size)
         if not isinstance(img, Tensor):
             img = F.pil_to_tensor(img)
         img = F.convert_image_dtype(img, torch.float)
-        return self._normalize(img)
+        img = F.normalize(img, mean=self._mean, std=self._std)
+        return img
```
Comment (datumbox) [inline torchvision/prototype/transforms/_presets.py:44]: replacing with the functional equivalents.

File: torchvision/prototype/transforms/_presets.py [hunk 3]
```diff
@@ -26,42 +25,47 @@ def __init__(
...
     def forward(self, vid: Tensor) -> Tensor:
         vid = vid.permute(0, 3, 1, 2)  # (T, H, W, C) => (T, C, H, W)
-        vid = self._convert(vid)
-        vid = self._resize(vid)
-        vid = self._normalize(vid)
-        vid = self._crop(vid)
+        vid = F.resize(vid, self._size, interpolation=self._interpolation)
+        vid = F.center_crop(vid, self._crop_size)
+        vid = F.convert_image_dtype(vid, torch.float)
+        vid = F.normalize(vid, mean=self._mean, std=self._std)
```
Comment (datumbox) [inline torchvision/prototype/transforms/_presets.py:68]: I reordered the operations. This is more "usual/canonical" order of ops. I'm running tests to confirm the accuracy remains the same.

**Review by datumbox — COMMENTED** [review]
Few clarifications to assist review:

File: torchvision/prototype/transforms/_presets.py (hunk 3 above)
Comment (datumbox) [reply torchvision/prototype/transforms/_presets.py:68]: The accuracy before and after the change remains the same. The above change is safe.

File: torchvision/prototype/models/video/resnet.py (hunk 1 above)
Comment (pmeier) [reply torchvision/prototype/models/video/resnet.py:65]: This is just for appearance, right?

File: torchvision/prototype/transforms/_presets.py [hunk 4]
```diff
@@ -26,42 +25,47 @@ def __init__(
...
+        img = F.normalize(img, mean=self._mean, std=self._std)
+        return img
 
 
 class Kinect400Eval(nn.Module):
     def __init__(
         self,
-        resize_size: Tuple[int, int],
         crop_size: Tuple[int, int],
+        resize_size: Tuple[int, int],
```
Comment (pmeier) [inline torchvision/prototype/transforms/_presets.py:51]: Unrelated to this PR, but why do we use `Tuple[int, int]` here for the sizes, but plain `int`'s for `ImageNetEval`?

**Review by pmeier — APPROVED** [review]
Two questions inline, otherwise LGTM. Thanks Vasilis!

File: torchvision/prototype/transforms/_presets.py (hunk 2 above)
Comment (datumbox) [reply torchvision/prototype/transforms/_presets.py:44]: It's equivalent. Confirmed by checking the accuracy of models before and after.

File: torchvision/prototype/models/video/resnet.py (hunk 1 above)
Comment (datumbox) [reply torchvision/prototype/models/video/resnet.py:65]: yes, just for styling. :)

File: torchvision/prototype/transforms/_presets.py (hunk 4 above)
Comment (datumbox) [reply torchvision/prototype/transforms/_presets.py:51]: This is a good point. This is how the original recipes have been implemented, so I need to do this here too. If you see the resize operates differently if you specify both dimensions vs only 1. Prior merging we should consider adding unions etc here and cleaning up further.
